### PR TITLE
Allsky publishdata

### PR DESCRIPTION
**Pull request by TheCrowsNestAstro**
Module that can publish AllSky data to Redis, MQTT or REST POST  ALLSKY_EXTRA data will be included as well.

### CODE REVIEW

File: allsky_publishdata/requirements.txt.py
Comment (Alex-developer) [inline allsky_publishdata/requirements.txt.py:1]: This file should be a .txt file not a .py file

**Review by Alex-developer — COMMENTED** [review]
Looks good - Just one minor change t o the requirements file :-)

File: allsky_publishdata/requirements.txt.py
Comment (TheCrowsNestAstro) [reply allsky_publishdata/requirements.txt.py:1]: Oups! :)

**Comment by TheCrowsNestAstro** [comment]
Oups! Renamed!

**Review by Alex-developer — APPROVED** [review]

**Pull request merged**